avro/avro.go

```diff
@@ -0,0 +1,51 @@
+// Package avro provides a user functionality to return the
+// avro encoding of s.
```

Comment:
Can the comment be updated or removed please? What's "encoding of s"?

Reply:
These are comments left over from the original library in chs.go - don't think they add any value so will remove